src/harvest-rpde.js

```diff
@@ -113,7 +119,7 @@ async function harvestRPDE({
       if (rpdeValidationErrors.length > 0) {
         if (multibar) multibar.stop();
         logError(`\nFATAL ERROR: RPDE Validation Error(s) found on RPDE feed ${feedContextIdentifier} page "${url}":\n${rpdeValidationErrors.map(error => `- ${error.message.split('\n')[0]}`).join('\n')}\n`);
-        process.exit(1);
+        onError();
```

Comment:
Should `harvestRPDE(..)` also return after each of these `onError()` calls?

Also, ideally, the argument to `onError()` should provide some context over what went wrong but that sounds like a thing for later. Maybe a TODO?

Reply:
Yeh done